tools/firtool/firtool.cpp

```diff
@@ -536,6 +547,15 @@ processBuffer(MLIRContext &context, TimingScope &ts, llvm::SourceMgr &sourceMgr,
   if (inferResets)
     pm.nest<firrtl::CircuitOp>().addPass(firrtl::createInferResetsPass());
 
+  if (exportChiselInterface) {
+    if (chiselInterfaceOutDirectory == "-") {
```

Comment:
Nit: No braces around single-statement if/else blocks, so no braces needed around both of the `pm.nest...` lines.

Reply:
If I remove any of the braces in these lines, I get the following warning:
> Add explicit braces to avoid dangling else